extensions/wikia/AdEngine/js/provider/gpt/helper.js

```diff
@@ -196,7 +198,12 @@ define('ext.wikia.adEngine.provider.gpt.helper', [
 	}
 
 	function refreshTargetingData(slot) {
-		slot.setTargeting('uap', uapContext.getUapId().toString());
+		var el = doc.getElementById(slot.getSlotElementId()),
+			gptParams = JSON.parse(el.getAttribute('data-gpt-slot-params'));
```

Comment:
Please, don't rely on targeting in HTML attribute. It's a side effect of our debugging tools.

Reply:
fixed

Reply:
Why do we need to read targeting from DOM node? Can't we get it from a slot?